brainspace/plotting/surface_plotting.py

```diff
@@ -532,10 +537,18 @@ def plot_hemispheres(surf_lh, surf_rh, array_name=None, color_bar=False,
                 array_name2.append(an)
         array_name = np.asarray(array_name2)[:, None]
 
+    if layout_style == 'grid':
+        array_name = np.full((2, 2), fill_value=array_name[0][0]) 
+        layout = np.array(layout).reshape(2, 2).T.tolist()
+        view = view = [['lateral', 'medial'], ['medial', 'lateral']]
+        share = 'both'
+    else:
+        share = 'r'
```

Comment:
```suggestion
        view = ['lateral', 'medial', 'lateral', 'medial']
        share = 'r'
```

Move view here and remove line 519